packages/excalidraw/change.ts

```diff
@@ -1457,10 +1459,13 @@ export class ElementsChange implements Change<SceneElementsMap> {
   private static redrawBoundArrows(
     elements: SceneElementsMap,
     changed: Map<string, OrderedExcalidrawElement>,
+    scene: Scene,
   ) {
     for (const element of changed.values()) {
       if (!element.isDeleted && isBindableElement(element)) {
-        updateBoundElements(element, elements);
+        updateBoundElements(element, elements, scene, {
```

Comment:
Do you see a conceptual problem with passing scene into these functions? I proposed passing scene because of performance reasons (not creating an array of elements besides the ElementsMap on every elbow arrow change frame), expecting that there might be more needed from scene down the line.

Reply:
I just spent 30 mins looking into it and there are ElemenentsMap branded type clashes, which need to be checked thoroughly before we can move forward with it. I committed my attempt at removing scene arguments in a separate branch for later.